package.json

```diff
@@ -1,30 +1,20 @@
 {
   "name": "lemmy-js-client",
-  "version": "0.19.2-alpha.3",
   "description": "A javascript / typescript client for Lemmy",
-  "repository": "https://github.com/LemmyNet/lemmy-js-client",
-  "license": "AGPL-3.0",
+  "version": "0.19.2-alpha.3",
```

Comment:
Out of scope with this PR, but how should we handle versioning for the JS client when it hasn't changed between the others repos version upgrade?

Reply:
Really we should upgrade it either right before or after version release.